src/Uno.UI/UI/Xaml/Controls/Popup/PopupPanel.cs

```diff
@@ -241,12 +241,19 @@ private protected override void OnUnloaded()
 	private void OnPointerPressed(object sender, PointerRoutedEventArgs args)
 	{
 		// Make sure we are the original source.  We do not want to handle PointerPressed on the Popup itself.
-		if (args.OriginalSource == this && Popup is { } popup
-		)
+		if (args.OriginalSource == this && Popup is { } popup)
 		{
+			// CommandBars in WinUI don't rely on IsLightDismissEnabled, instead there's IsSticky.
+			// Instead of handling it here, CommandBar should handle it using an LTE (look at the comment
+			// in AppBar.SetupOverlayState) but we don't have the logic implemented in Uno yet, so we
+			// rely on this workaround to close CommandBar's popup.
+			if (popup.TemplatedParent is CommandBar cb)
```

Comment:
Looks good, are you able to make a UI test for it? Runtime may be tricky.

Reply:
Not a big a fan of specific cases like this which are tightly coupling unrelated classes ... but acceptable as a **temp** workaround :/